src/Warnings.hpp

```diff
@@ -142,6 +142,7 @@ THE SOFTWARE.
     _Pragma("clang diagnostic ignored \"-Wunused-parameter\"") \
     _Pragma("clang diagnostic ignored \"-Wused-but-marked-unused\"") \
     _Pragma("clang diagnostic ignored \"-Wweak-vtables\"") \
+    _Pragma("clang diagnostic ignored \"-Wreserved-identifier\"") \
```

Comment:
Isn't this a new warning? Since we still want to support older compiler, I think you have to do the `__has_warning` dance as with other warnings below, too. I.e. something like:
```c++
  #if __has_warning("-Wreserved-identifier")
    #define ABL_PRAGMA_CLANG_DIAGNOSTIC_IGNORED_RESERVED_IDENTIFIER \
    _Pragma("clang diagnostic ignored \"-Wreserved-identifier\"")
  #else
    #define ABL_PRAGMA_CLANG_DIAGNOSTIC_IGNORED_RESERVED_IDENTIFIER
  #endif
```
and then use that macro here:
```c++
...
_Pragma("clang diagnostic ignored \"-Wweak-vtables\"") \
ABL_PRAGMA_CLANG_DIAGNOSTIC_IGNORED_RESERVED_IDENTIFIER \
ABL_PRAGMA_CLANG_DIAGNOSTIC_IGNORED_COMMA \
...
```

Reply:
Ok, @ala-ableton wa quicker than me. 🤷